### DISCUSSION TITLE
Possible error in the Apex and Antapex points?

### DISCUSSION THREAD

**Discussion by Ironman65** [Stellarium/stellarium#961] [Q&A] [answered]
If you check the Apex points box in the markers section. It displays the Antapex point in the constellation Aries, if I’m not mistaken, the Antapex point should be in the Canis Major / Columbia area of space. While the Apex point is displayed in the constellation Libra near the star alpha Libra. It should be displayed in the Hercules/ Lyra of space. If I’m wrong please disregard if so.

**Comment by gzotti** [comment, marked as answer]
What does "Apex" mean for you? 

Hold the mouse cursor above the Apices checkbox to read what it means in this program. 

The interstellar "solar apex" is irrelevant in the context of this program.

**Comment by Ironman65** [comment]
Mr. Gzotti. Thanks for the tips on using the mouse for this information. You’re correct! Based on what you posted. I always based this terminology on the Sun’s point of view and not the Earth’s. So just disregard this post. Stellarium is one of the few programs that are capable teaching you such as my friend’s son all the way to the use by professional astronomers, and it run’s quite well even on older machines. Thanks’ for all that you do.
